temba/flows/views.py

```diff
@@ -2127,6 +2151,20 @@ def get(self, request, *args, **kwargs):
             flow = self.get_object(self.get_queryset())
             (active, visited) = flow.get_activity()
 
+            # update expiration time of editing for active editor
+            password = getattr(settings, "REDIS_PW", None)
+            flow_editors_cache = redis.Redis(
```

Comment:
You can use the method `get_redis_connection` importing `from django_redis import get_redis_connection` to get the default Redis connection established on the settings file. E.g. https://github.com/greatnonprofits-nfp/rapidpro/blob/master/temba/campaigns/tasks.py#L25